OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -287,9 +287,9 @@ public void onFragmentSelected() {
         }
         // Add important dates tab
         if (environment.getConfig().isCourseDatesEnabled()) {
-            items.add(new FragmentItemModel(CourseDatesFragment.class,
```

Comment:
Now CourseDatesFragment.java is unused, I think we can remove this.

@miankhalid thoughts?

Reply:
I'll remove it

Reply:
Yep, nuke it!